### ISSUE TITLE
My thanks ! Your mod is amazing

### ISSUE THREAD

**Issue by Silka64** [MrNavaStar/InvSync#68]
I just wish you my thanks for this mod and the big work on him.

Im using it on survival server with one server is using terralith as world gen datapack and the other using William's overhauled. It's just perfect !

**Comment by MrNavaStar** [comment]
I'm glad you like it! 

I am currently working on a rewrite for 1.20.4 +. Hopefully I have something to show for it in the next few months!

**Comment by Silka64** [comment]
That sounds good! Preparing a 1.21 survival adventure based on 3 servers for me and some friends.

First: mods + Terralith (classic size biome).
Second: mods + Terralith (Insane Large Biome) + specific mod allowing travelling nether not by 1:8  ratio but by 1:64.
Last one: mods + vanilla gen.

Using velocity + FabricProxyLite as proxy server and a mod i code to travelling between servers with command.

One question : What if im using different mods adding items, advancement, etc. between server ? The items are just ignored by invSync without causing any issues ?

Without your mod, this wouldn't be possible! Thank you so much

**Comment by MrNavaStar** [comment]
Modded items/advancements/stats/effects should all sync out of the box

**Comment by MrNavaStar** [comment]
Idk what happens with different mods on each server, I've never tested that.

**Comment by Silka64** [comment]
Yeah, because im thinking about using a server with BetterEnd mod and if i switch from server1(BetterEnd=yes) to server2(BetterEnd=no) while having betterEnd Item on the inventory.... 

If then later im back from server2 to server1 and my inventory changed, maybe can occur a bug. 

Maybe a solution would be to detect unkown item and then restitute them on the ground when the player come back. scuse me, very bad english
